docs/migrating-v1beta1-to-v1.md

```diff
@@ -74,6 +74,15 @@ to `Pipeline`s and `Task`s defined in other namespaces in the cluster.
 ```yaml
 apiVersion: tekton.dev/v1beta1
 kind: TaskRun
+metadata:
+  name: cluster-task-reference
+spec:
+  taskRef:
+    name: example-task
+    kind: ClusterTask
+---
+apiVersion: tekton.dev/v1
```

Comment:
I'd suggest:

before:
```yaml
apiVersion: tekton.dev/v1beta1
kind: TaskRun
metadata:
  name: cluster-task-reference
spec:
  taskRef:
    name: example-task
    kind: ClusterTask
```
After:
...

This example makes it look like you need to use both at the same time

Reply:
same for the bundle one above